tests/validation/cattlevalidationtest/core/test_rancher_compose.py

```diff
@@ -0,0 +1,401 @@
+from common_fixtures import *  # NOQA
+
+TEST_SERVICE_OPT_IMAGE = 'ibuildthecloud/helloworld'
+TEST_SERVICE_OPT_IMAGE_LATEST = TEST_SERVICE_OPT_IMAGE + ':latest'
+TEST_SERVICE_OPT_IMAGE_UUID = 'docker:' + TEST_SERVICE_OPT_IMAGE_LATEST
+LB_IMAGE_UUID = "docker:sangeetha/testlbsd:latest"
+
+logger = logging.getLogger(__name__)
+
+
+def test_rancher_compose_service(super_client, client,
+                                 rancher_compose_container,
+                                 socat_containers):
+
+    vol_container = client.create_container(imageUuid=TEST_IMAGE_UUID,
+                                            name=random_str(),
+                                            labels={"c1": "vol"}
+                                            )
+    vol_container = client.wait_success(vol_container)
+
+    volume_in_host = "/test/container"
+    volume_in_container = "/test/vol1"
+    docker_vol_value = volume_in_host + ":" + volume_in_container + ":ro"
+
+    cap_add = ["CHOWN"]
+    cap_drop = ["KILL"]
+    restart_policy = {"maximumRetryCount": 10, "name": "on-failure"}
+    dns_search = ['1.2.3.4']
+    dns_name = ['1.2.3.4']
+    domain_name = "rancher.io"
+    host_name = "test"
+    user = "root"
+    command = ["sleep", "9000"]
+    env_var = {"TEST_FILE": "/etc/testpath.conf"}
+    memory = 8000000
+    cpu_set = "0"
+    cpu_shares = 400
+    # Not including "dataVolumesFrom": [vol_container.id] since it is not
+    # implemented yet
+
+    launch_config = {"imageUuid": TEST_SERVICE_OPT_IMAGE_UUID,
```

Comment:
@sangeethah do we have a separate test case for validating "net" option? If not, we should include the validation here.